upstream_sync.py

```diff
@@ -24,10 +24,24 @@
 tmp_dir = '/var/tmp/upstream_sync-{0}'.format(user)
 
 
-def build_yum_config(name, url, sslcacert, sslcert, sslkey):
+def make_dir(dir_path, mode=0700):
+    """ checks if a directory exists and create it if necessary
+    :param mode: mode for makedirs
+    :param dir_path: dir path for check
+
+    :return: None
+    """
+    if not os.path.isdir(dir_path):
+        try:
+            os.makedirs(dir_path, mode)
+        except IOError as e:
+            print e
+            sys.exit(1)
+
+
+def build_yum_config(name, url, sslcacert, sslcert, sslkey, exclude):
     # Check tmp path exist
-    if not os.path.isdir(tmp_dir):
-        os.mkdir(tmp_dir)
+    make_dir(tmp_dir)
```

Comment:
You're right, I need to be a little easier when I see a * .pem or * .key in the configuration files. Current ```umask``` value should be a right decision.